LunaDll/LuaMain/LuaProxyFFI.cpp

```diff
@@ -768,6 +771,41 @@ typedef struct ExtendedBlockFields_\
     FFI_EXPORT(float) LunaLuaLegacyRNGGenerateNumber() {
         return VB6RNG::generateNumber();
     }
+
+    FFI_EXPORT(bool) LunaLuaIsRecordingGIF()
+    {
+        return g_GLEngine.GifRecorderIsRunning();
+    }
+    
+    FFI_EXPORT(bool) LunaLuaIsFullscreen()
+    {
+        if (gMainWindowHwnd != NULL)
+        {
+            WINDOWPLACEMENT wndpl;
+            wndpl.length = sizeof(WINDOWPLACEMENT);
+            if (GetWindowPlacement(gMainWindowHwnd, &wndpl))
+            {
+                if (wndpl.showCmd == SW_MAXIMIZE) {
```

Comment:
As per the previous comment https://github.com/WohlSoft/LunaLua/pull/51#discussion_r1186727505 _(marked 'outdated' by Github because it lost track of where the code was)_:

Any reason why the `if` statement here, rather than just writing `return (wndpl.showCmd == SW_MAXIMIZE);`?